client/web/src/site-admin/overview/SiteAdminOverviewPage.tsx

```diff
@@ -45,8 +45,8 @@ interface Props extends ActivationProps, ThemeProps {
 const fetchOverview = (): Observable<{
     repositories: number | null
     repositoryStats: {
-        gitDirBytes: number
-        indexedLinesCount: number
+        gitDirBytes: string
+        indexedLinesCount: string
```

Comment:
There should be type aliases generated for this (to make the semantic clearer).
```suggestion
        gitDirBytes: Scalars['BigInt']
        indexedLinesCount: Scalars['BigInt']
```